packages/fluent-theme/src/components/assets/SlidingDots.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { memo, useMemo } from 'react';
+
+type SlidingDotsProps = Readonly<{
+  className?: string | undefined;
+}>;
+
+function SlidingDots({ className }: SlidingDotsProps) {
+  const id = useMemo(() => `webchat__sliding-dots-asset--${crypto.randomUUID()}`, []);
```

Comment:
Suggesting an  `object` element with src set instead as it is both: more performant and allows start/stop the animation.

For checking the particular media query suggesting using `window.matchMedia`.

Reply:
Update: seems we can't use object because it restricts accessing the content

Reply:
Using blob: URL now.